gear_ratios: 
  - 161
  - 161
  - 121
  - 161
  - 161
  - 101
zero_points: 
  - 21075821
  - 3787515
  - -30318
  - -366005
  - 11787161
  - 40952066
enc_resolution: 131072
a:
  - 161
  - 161
  - 121
  - 161
  - 161
  - 101
alpha:
  - 161
  - 161
  - 121
  - 161
  - 161
  - 101
d:
  - 161
  - 161
  - 121
  - 161
  - 161
  - 101
